### DISCUSSION TITLE
change a pin to different one

### DISCUSSION THREAD

**Discussion by mikimause9** [darthcloud/BlueRetro#828] [Q&A]
I have kind of an embarrassing problem. I flashed my esp32-s with BlueRetro hw2 for internal installation. In a testing process I... ripped the pin 35. As it turned out it is for Controller port 1 detection with Gamecube. And now I cannot make BlueRetro assign Bluetooth controller to port 1. Is there any way to change pin 35 to a different unused one?

**Comment by konwektor** [comment]
For example change reset pin 14 with detect controller 1 pin35 in : /main/system/manager.c
line 30 and 38.
I am not sure is there are any extra conditions to checking those pins. Worth to try.

`/*
 * Copyright (c) 2019-2022, Jacques Gagnon
 * SPDX-License-Identifier: Apache-2.0
 */

#include <stdio.h>
#include <string.h>
#include <freertos/FreeRTOS.h>
#include <freertos/task.h>
#include <freertos/ringbuf.h>
#include <esp_partition.h>
#include <esp_sleep.h>
#include <esp_system.h>
#include "driver/gpio.h"
#include "hal/ledc_hal.h"
#include "hal/gpio_hal.h"
#include "driver/ledc.h"
#include "esp_rom_gpio.h"
#include "adapter/adapter.h"
#include "bluetooth/host.h"
#include "bluetooth/hci.h"
#include "wired/wired_comm.h"
#include "tools/util.h"
#include "system/fs.h"
#include "system/led.h"
#include "manager.h"

#define BOOT_BTN_PIN 0

#define RESET_PIN 14

#define POWER_ON_PIN 13
#define POWER_OFF_PIN 16
#define POWER_SENSE_PIN 39

#define POWER_OFF_ALT_PIN 12

#define SENSE_P1_PIN 35
#define SENSE_P2_PIN 36
#define SENSE_P3_PIN 32
#define SENSE_P4_PIN 33
`
